.github/CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+/.bundle/ @cockroachdb/docs-infra-prs
+/.github/ @cockroachdb/docs-infra-prs
+/__tests__/ @cockroachdb/docs-infra-prs
+/_layouts/  @cockroachdb/docs-infra-prs
+/_plugins/  @cockroachdb/docs-infra-prs
+/archived/  @cockroachdb/docs-infra-prs
```

Comment:
I don't think this should be on `docs-infra-prs`

Reply:
Fair. Will remove.